Examples/SessionTools-macOSExample/HomeViewController.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  ViewController.swift
+//  SessionTools-macOSExample
+//
+//  Copyright (c) 2019 Bottle Rocket Studios. All rights reserved.
+//
+
+import Cocoa
+
+class HomeViewController: NSViewController {
+    
+    // MARK: - Properties
+    
+    @IBOutlet private var helloLabel: NSTextField!
```

Comment:
Since this is a text field, it can be renamed helloTextField.
```
@IBOutlet private var helloTextField: NSTextField!
```

Reply:
Adjusted name.